src/components/SearchBar/index.tsx

```diff
@@ -114,12 +114,13 @@ const SearchBar: React.FC<SearchBarProps> = ({
       <StyledInput
         ref={inputRef}
         aria-label="searchbar-input"
+        aria-labelledby="searchbar-input"
```

Comment:
I guess it's not great that we have both, but I need `aria-label` to avoid this console error and I need `aria-labelledby` to avoid a TypeScript error from LG

<img width="588" alt="Screen Shot 2022-10-28 at 3 34 24 PM" src="https://user-images.githubusercontent.com/47064971/198725426-6d139155-5512-420b-971a-4cad3a4e9612.png">